docs/adrs/0002-field-for-data-type-form-information.md

```diff
@@ -0,0 +1,184 @@
+    * https://opengeometadata.org/ogm-aardvark/#resource-type-values-loc (e.g. "Atlases", "Cadastral maps", "Quadrangle maps")
+    * https://opengeometadata.org/ogm-aardvark/#resource-type-values-ogm (e.g. "Polygon", "Raster", "Line", "Point")
+
+### Possible Solutions
+
+#### Option 1- Use `subjects` with `kind="Data Type"`
+
+In this approach, "Data Type" values would be stored as `subjects` with `kind="Data Type"`.
+
+Example:
+```json
+{
+    "subjects": [
+        {
+            "value": "Polygon",
+            "kind": "Data Type"
+        },
+        {
+            "value": "Vector",
+            "kind": "Data Type"
+        }
+    ]
+}
+```
+
+Pros:
+  * does not require a change to TIMDEX data model anywhere
+
+Cons:
+  * these "Data Type" values don't feel like subjects; they are not really _about_ the resource so much as describing its type/structure/form 
+
+
+#### Option 2- Create new, multivalued string field `form`
+
+In this approach, "Data Type" value would be stored in a new, multivalued string field `form`:
+
+Example:
+```json
+{
+    "form": ["Polygon", "Vector"]
+}
+```
+
+Pros:
+  * purely additive change to data model
+  * simple, top level property makes aggregations very simple
+
+Cons:
+  * still require, and sit along next to, `literary_form` field for describing text sources as "Fiction" or "Nonfiction"
+
+
+#### Option 3- Create new, multivalued objects field `form`; collapse `literary_form` into this
+
+In this approach, "Data Type" value would be stored in a new, multivalued object field `form`:
+
+Example:
+```json
+{
+    "form": [
+        {
+            "value": "Polygon",
+            "kind": "Data Type"
+        },
+        {
+            "value": "Vector",
+            "kind": "Data Type"
+        }
+    ]
+}
+```
+
+Pros:
+  * allows collapsing of `literary_form` field; noting some shared sentiment that this field might be too source-specific for TIMDEX
+  * like other object fields, leaves the door open for adding a `uri` property at a later time
+
+Cons:
+  * would require reworking the transformations + re-indexing any sources that use `literary_form`
+  * nested field type, a bit harder to query for aggregations
+
+#### Option 4 - Use `file_formats` for current `format` values and `format` for Data Type values
+
+In this approach, the current `MITAardvark.format` values would shift to the previously unused `MITAardvark.file_formats` property and the Data Type values would be stored in `MITAardvark.format`
+
+Example:
+```json
+
+{
+    "content_type": "Geospatial data",
+    "format": ["Polygon", "Point", "Raster", "Image"],
+    "file_formats": ["Shapefile", "TIFF", "GeoTIFF", "JPEG"]
+}
+```
+
+Pros:
+  * does not require TIMDEX data model changes
+
+Cons:
+  * `file_formats` has previously only stored MIME type values, such as `application/pdf`
+  * may require explanation of the facet mapping in the UI documentation
+  * may require updates of other transform classes for consistency
```

Comment:
I'm finding Option #4 pretty appealing now.  

One question: what if we had a record for a print map?  If the Aardvark record had `dct_format_s="Print map"`, are we okay with this value mapping to `file_formats`?  

Maybe helpful for discussion, [this spreadsheet](https://docs.google.com/spreadsheets/d/10oNtwD-3HagVHFfa00sB8eWs5EVmmWMiLSq2q08VU-w/edit#gid=1635328382) has `dct_format_s` and `gbl_resourceType_sm` values across all MIT and OGM records.

In support of Option #4, I'm hard pressed to find a value that _isn't_ a digital file format in essence (where even msot of the complex ones could be mapped to "Online resource" or something similar).

Reply:
Another question: if we go this route, are we okay losing mimetype as a value in a TIMDEX record?  My inclination is that it would be okay as I'd imagine this field is not widely used yet.

If we did want specifics about mimetypes, perhaps that could get added to the `links` section?  Seems more meaningful for a mimetype to point _precisely_ to an actual resource vs the record as an abstract whole.

Reply:
> If we did want specifics about mimetypes, perhaps that could get added to the `links` section? Seems more meaningful for a mimetype to point _precisely_ to an actual resource vs the record as an abstract whole.

Agree that including it with the actual download link would be more useful

Reply:
> One question: what if we had a record for a print map? If the Aardvark record had `dct_format_s="Print map"`, are we okay with this value mapping to `file_formats`?

If we do go this route, we may want to change the name from `file_formats` so it's more broadly applicable but I don't have a suggestion yet

Reply:
> > One question: what if we had a record for a print map? If the Aardvark record had `dct_format_s="Print map"`, are we okay with this value mapping to `file_formats`?
> 
> If we do go this route, we may want to change the name from `file_formats` so it's more broadly applicable but I don't have a suggestion yet

I've had time to look more closely at the spreadsheet linked above, and I don't think we'll encounter "Print Maps", at least in the `gismit` or `gisogm` sources.  So maybe this is moot?  I think it's safe to say that the values we get back from `dct_format_s` in the MITAardvark files will uniformly be digital in nature.  And therefore, `file_formats` could be applicable.

Reply:
> > If we did want specifics about mimetypes, perhaps that could get added to the `links` section? Seems more meaningful for a mimetype to point _precisely_ to an actual resource vs the record as an abstract whole.
> 
> Agree that including it with the actual download link would be more useful, `Link.file_type` attribute?

If we did repurpose `file_formats`, and therefore needed to move those mimetype data points to another field, and we _did_ like putting them on the `links` entries, I might propose `mimetype` explicitly; no ambiguity there.

If we go that route, I think we'd want to make sure the links made sense though.  If we're just pointing someone back to an Alma record, or ASpace page, I'm not sure that an `link.mimetype=application/pdf` makes sense.